Projects/Team1/Team1_Wpf/Team1_WpfApp/Team1_WpfApp/Fairytale/IMaterial.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Team1_WpfApp.Fairytale
+{
+    public interface IMaterial
```

Comment:
Do you use this interface? I haven't found a code that calls it.

Reply:
Yes, I used this interface
namespace Team1_WpfApp.Fairytale
{

    public class Material : IMaterial
    {
        private string name;
        private int strength;

        public int Strength
        {
            get
            {
                return this.strength;
            }
            set
            {
                this.strength = value;
            }
        }

Reply:
I mean that you don't call this interface like this:
   SomeObject.SomeMethod(IMaterial someMaterial){.....}

And if you don't use this interface in methods implementations it means that we no need this interface at all. In our case, it doesn't give us any privileges but only the redundant code.